state/indexer/block/kv/kv.go

```diff
@@ -86,6 +115,24 @@ func (idx *BlockerIndexer) Index(bh types.EventDataNewBlockEvents) error {
 	return batch.WriteSync()
 }
 
+func (idx *BlockerIndexer) Prune(retainHeight int64) {
+	if retainHeight < idx.lbRetainedHeight {
```

Comment:
This should be retrieved from the Database if it is not yet set.

Reply:
I retrieve it from the DB in the `func New(store dbm.DB) *BlockerIndexer`, inside `getLBRetainHeight`. 

My logic was to retrieve it a single time at the beginning and then operate with the in-memory value.